apps/frontend/src/types/FormData.d.ts

```diff
@@ -0,0 +1,14 @@
+// This will need to be updated to match the shape of the data we receive from the backend
```

Comment:
this is the current schema from the backend https://github.com/Code-4-Community/constellation/blob/main/apps/backend/src/schema/schema.ts